### PR TITLE
LPS-68765 Tooltip refresh

### PR DESCRIPTION
**Pull request by jbalsas**

### CODE REVIEW

File: modules/apps/foundation/frontend-editor/frontend-editor-alloyeditor-web/src/main/resources/META-INF/resources/alloyeditor.jsp [hunk 1]
```diff
@@ -38,7 +38,6 @@ String onChangeMethod = (String)request.getAttribute("liferay-ui:input-editor:on
 String onFocusMethod = (String)request.getAttribute("liferay-ui:input-editor:onFocusMethod");
 String onInitMethod = (String)request.getAttribute("liferay-ui:input-editor:onInitMethod");
 String placeholder = GetterUtil.getString((String)request.getAttribute("liferay-ui:input-editor:placeholder"));
-String required = (String)request.getAttribute("liferay-ui:input-editor:required");
```
Comment (Robert-Frampton) [inline modules/apps/foundation/frontend-editor/frontend-editor-alloyeditor-web/src/main/resources/META-INF/resources/alloyeditor.jsp]: Hey @antoniopol06 

Is this a part of the tooltip fix? It seems that might be unrelated

File: modules/apps/foundation/frontend-editor/frontend-editor-alloyeditor-web/src/main/resources/META-INF/resources/alloyeditor.jsp [hunk 2]
```diff
@@ -234,7 +237,6 @@ name = HtmlUtil.escapeJS(name);
 			plugins.push(
 				{
 					cfg: {
-						uploadItemReturnType: '<%= editorOptions.getUploadItemReturnType() %>',
```
Comment (Robert-Frampton) [inline modules/apps/foundation/frontend-editor/frontend-editor-alloyeditor-web/src/main/resources/META-INF/resources/alloyeditor.jsp]: Same question here, is this related?

Comment (antoniopol06) [inline modules/apps/foundation/frontend-editor/frontend-editor-alloyeditor-web/src/main/resources/META-INF/resources/alloyeditor.jsp]: Hi @Robert-Frampton !

You are right, sorry! this part of code and the previous one are not related to this issue.

**Comment by Robert-Frampton** [comment]
Hey @antoniopol06 

Can you make those changes and resend? I'll close this one since this is from Chema's branch. Thanks!
